homeassistant/components/sensor/darksky.py

```diff
@@ -262,6 +291,9 @@ def get_state(self, data):
         if state is None:
             return state
 
+        if "summary" in self.type:
```

Comment:
Nope, since in DarkSky the sensor supports several types that are forecasts: `summary`, `minutely_summary`, `hourly_summary`, and `daily_summary`. I think `"string" in variable` is more readable/Pythonic than `variable.find("string") >= 0`.